src/index.html

```diff
@@ -11,8 +11,57 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        src="images/imac.jpeg"
+        alt="iMac A1419"
+        class="card__image"
+      />
+      <div class="card__info info">
+        <h1 class="info__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h1>
+        <h3 class="info__code">Product code: 195434</h3>
+      </div>
+      <div class="reviews">
+        <div class="stars-1">
+          <div class="stars-1__star"></div>
```

Comment:
```suggestion
        <div class="stars stars--4">
          <div class="stars__star"></div>
```
rewrite from stars task